addons/web/static/src/views/form/form_status_indicator/form_status_indicator.xml

```diff
@@ -3,19 +3,37 @@
 
     <t t-name="web.FormStatusIndicator" owl="1">
         <div class="o_form_status_indicator d-flex align-self-center align-items-center ms-4 rounded-pill position-relative" t-att-class="{ o_form_status_indicator_new_record: props.model.root.isVirtual }">
+            <div class="o_form_status_indicator_buttons d-flex" t-att-class="{ invisible: !(props.model.root.isVirtual or displayButtons) }">
+                <button
+                    type="button"
+                    class="o_form_button_save btn btn-light btn-sm"
+                    t-att-disabled="props.isDisabled"
+                    data-hotkey="s"
+                    t-on-click.stop="save"
+                    data-tooltip="Save manually"
+                    aria-label="Save manually">
+                    <i class="fa fa-cloud-upload fa-fw" />
+                </button>
+                <button
+                    type="button"
+                    class="o_form_button_cancel btn btn-light btn-sm"
+                    t-att-disabled="props.isDisabled"
+                    data-hotkey="j"
+                    t-on-click.stop="discard"
+                    data-tooltip="Discard changes"
+                    aria-label="Discard changes"                    >
```

Comment:
that spacing is ________________________long________________________ before the `>` :grin:

Reply:
![image](https://user-images.githubusercontent.com/9530767/194550627-fa125ea1-bd4f-4c5a-80fa-c3ef10b2d879.png)